drop platform exclusions for coreos.ignition.security.tls

https://github.com/coreos/coreos-assembler/blob/d3c7ec094a025fa3e8c9217f4e58245fb06b80d2/mantle/kola/tests/ignition/security.go#L63-L65

The existing exlcusions of `do`, `packet`, and `qemu` should be dropped. The linked issues are closed and according to @bgilbert:

> I believe the kernel restructured how it handles early entropy, and on DO we're currently limited to custom images with DHCP.  so I'd say we can drop all of those exclusions.

We should drop these exclusions and verify the test executes successfully in QEMU. We don't currently run automated tests against Packet and DO so we can test on those platforms when we start running tests again.

Htting a problem with the IP address being set in the test:

```
======== Re-running failed tests (flake detection) ========

=== RUN   coreos.ignition.security.tls
--- FAIL: coreos.ignition.security.tls (24.95s)
        cluster.go:192: "sudo bash -c 'openssl req -new -x509 -sha256 -key /var/tls/server.key -out /var/tls/server.crt -days 3650 -subj \"/CN=127.0.0.1:34577\" -config <(cat <<-EOF\n[req]\ndefault_bits = 2048\ndefault_md = sha256\ndistinguished_name = dn\n\n[ dn ]\nCN = 127.0.0.1:34577\n\n[ SAN ]\nsubjectAltName = IP:127.0.0.1:34577\nEOF\n) -extensions SAN'" failed: Process exited with status 1; logs from journalctl -t kola:
+ sudo mkdir /var/tls
+ sudo openssl ecparam -genkey -name secp384r1 -out /var/tls/server.key
+ sudo bash -c 'openssl req -new -x509 -sha256 -key /var/tls/server.key -out /var/tls/server.crt -days 3650 -subj "/CN=127.0.0.1:34577" -config <(cat <<-EOF
[req]
default_bits = 2048
default_md = sha256
distinguished_name = dn

[ dn ]
CN = 127.0.0.1:34577

[ SAN ]
subjectAltName = IP:127.0.0.1:34577
EOF
) -extensions SAN'
Error Loading extension section SAN
140543589353280:error:220A4076:X509 V3 routines:a2i_GENERAL_NAME:bad ip address:crypto/x509v3/v3_alt.c:478:value=127.0.0.1:34577
140543589353280:error:22098080:X509 V3 routines:X509V3_EXT_nconf:error in extension:crypto/x509v3/v3_conf.c:47:name=subjectAltName, value=IP:127.0.0.1:34577
FAIL, output in tmp/kola/rerun
```

I am guessing reading the code that is expecting only the IP not the port. Getting only the port gets me a new error:
```
=== RUN   coreos.ignition.security.tls
--- FAIL: coreos.ignition.security.tls (623.42s)
        security.go:102: starting client: machine "c77f86e2-b390-4f11-922c-a9be11d73dbf" failed to start: ssh journalctl failed: ssh: handshake failed: read tcp 127.0.0.1:42910->127.0.0.1:45811: read: connection reset by peer
FAIL, output in tmp/kola

```
Trying to figure out what is happening here still.

--- a/mantle/kola/tests/ignition/security.go
+++ b/mantle/kola/tests/ignition/security.go
@@ -61,9 +61,8 @@ func init() {
 			"TLSServe": register.CreateNativeFuncWrap(TLSServe),
 		},
 		Tags: []string{"ignition"},
-		// DO: https://github.com/coreos/bugs/issues/2205
-		// Packet & QEMU: https://github.com/coreos/ignition/issues/645
-		ExcludePlatforms: []string{"do", "packet", "qemu"},
+		// QEMU unprivileged doesn't support multiple VMs communicating with each other.
+		ExcludePlatforms: []string{"qemu"},
 		Timeout:          20 * time.Minute,
 	})
 }

